### PR TITLE
ports some tiles form 1.0, revamps rigs, more bots!, bars and barred doors, random rig spawner!

### PR DESCRIPTION
**Pull request by blackwolf602**
ports some tiles form 1.0, revamps rigs, more bots!, bars and barred doors, marker shard toy, unitologist bible, lunch boxes. adds random rig spawner and ammo spawner currently only pulse shotgun and divet.

### CODE REVIEW

File: deadspace/code/modules/clothing/rigs/earthgov_faction/special_forces.dm [hunk 1]
```diff
@@ -1,25 +1,12 @@
 /datum/mod_theme/deadspace/special_forces
 	name = "Special Forces"
-	desc = "A heavily armoured rig suit, designed for military use, this one is highly advance, and typically used in Undisclosed Operations."
+	desc = "A military-grade RIG that was issued to the Earth Defense Force personnel. This one is highly advance, and typically used in Undisclosed Operations."
 	extended_desc = "Resource Integration Gear, or RIG for short is an integrated health management and strength augmentation system that assists users in previously impossible and dangerous environments, \
 		this suit is a staple across the galaxy for civilian applications. These suits are oxygenated, \
 		spaceworthy, resistant to fire and chemical threats."
```
Comment (DTraitor) [inline deadspace/code/modules/clothing/rigs/earthgov_faction/special_forces.dm:6]: Why did you remove most `extended_desc` but kept this one?

File: deadspace/code/modules/clothing/rigs/earthgov_faction/venture.dm [hunk 2]
```diff
@@ -6,20 +6,7 @@
 		spaceworthy, resistant to fire and chemical threats."
```
Comment (DTraitor) [inline deadspace/code/modules/clothing/rigs/earthgov_faction/venture.dm:6]: Ditto

File: deadspace/code/modules/clothing/rigs/patron_faction/maxstone.dm [hunk 3]
```diff
@@ -7,20 +7,7 @@
 		spaceworthy, resistant to fire and chemical threats."
```
Comment (DTraitor) [inline deadspace/code/modules/clothing/rigs/patron_faction/maxstone.dm:7]: Ditto

File: deadspace/code/modules/mob/living/simple_animal/bot/construction.dm [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+	icon = 'deadspace/icons/mob/animal48x48.dmi'
+	name = "incomplete cargobot assembly"
+	desc = "A frame with an leg attached to it."
+	icon_state = "cargobot_construction1"
+	created_name = "Cargobot"
+
+/obj/item/bot_assembly/cargobot/attackby(obj/item/W, mob/user, params)
+	..()
+	switch(build_step)
+		if(ASSEMBLY_FIRST_STEP)
```
Comment (DTraitor) [inline deadspace/code/modules/mob/living/simple_animal/bot/construction.dm:64]: Consider reworking these build steps into more generic code
E.g.
```
/generic_bot/attackby()
    switch(step)
        if(first_step)
            if(first_step())
                ++step

/generic_bot/specific_bot/first_step()
    return TRUE
```

**Review by DTraitor — CHANGES_REQUESTED** [review]

File: code/game/objects/items/apc_frame.dm [hunk 5]
```diff
@@ -34,3 +34,22 @@
...
+		..()
+
+/obj/item/wallframe/apc/attackby(obj/O, mob/user, params)
+	if(istype(O, /obj/item/bodypart/leg/left/robot) || istype(O, /obj/item/bodypart/leg/right/robot))
+		to_chat(user, span_notice("You add [O] to [src]."))
+		qdel(O)
+		qdel(src)
+		user.put_in_hands(new /obj/item/bot_assembly/cargobot)
+	else
+		..()
```
[the remark this answers is not in the document]
Comment (blackwolf602) [reply code/game/objects/items/apc_frame.dm:54]: these are two diffrent things

**Review by DTraitor — APPROVED** [review]
